tactics/ind_tables.ml

```diff
@@ -124,8 +124,17 @@ let define internal role id c poly univs =
   let ctx = UState.minimize univs in
   let c = UnivSubst.nf_evars_and_universes_opt_subst (fun _ -> None) (UState.subst ctx) c in
   let univs = UState.univ_entry ~poly ctx in
-  let entry = Declare.definition_entry ~univs c in
-  let kn, eff = Declare.declare_private_constant ~role ~kind:Decls.(IsDefinition Scheme) ~name:id (Declare.DefinitionEntry entry) in
+  let entry = {
```

Comment:
Why this change? Seems spurious.

Reply:
`definition_entry` has not the right type. I could export another function doing the same, though. WDYT?

Reply:
Likely that function already exists on my tree, let me see.

Reply:
Oh indeed I don't have it, I was waiting to see if `Evd.side_effects` would get a type parameter to actually match it. You can leave the code as-is if you want, and I'll introduce a `pure_definition_entry` in #10681 .